hw/ip/aes/dv/aes_base_sim_cfg.hjson

```diff
@@ -42,6 +42,19 @@
   // Default iterations for all tests - each test entry can override this.
   reseed: 50
 
+  build_modes: [
+    {
+      name: aes_no_masking
+      build_opts: ["+define+EN_MASKING=0",
+                   "+define+SBOX_IMPL=aes_pkg::SBoxImplLut" ]
```

Comment:
Could you please switch to `SBoxImplCanright` here? As discussed previously, `SBoxImplCanright` is what is used inside CSRNG for the ASIC. `SBoxImplLut` is used on FPGA only.